app/models/label_template.rb

```diff
@@ -1,17 +1,19 @@
 class LabelTemplate < ApplicationRecord
+  CLASS_TYPE_TEMPLATE_ALIASES = {
+    'TubeRack' => 'Plate',
+    'Tube' => 'SampleTube'
+  }.freeze
+
   validates_presence_of :name, :external_id
   validates_uniqueness_of :name, :external_id
 
-  def self.for_type(type, barcodetype = 'ean13')
-    type = {
-      'Plate' => ['TubeRack', 'Plate'],
-      'Tube' => ['Tube', 'SampleTube']
-    }.select { |k,v| v.include?(type) }.first[0]
+  def self.for_type(class_type, barcodetype = 'ean13')
```

Comment:
Rubocop performance cops didn't like the select.first, but this seemed an overall cleaner refactor.